open_academy/demo/demo.xml

```diff
@@ -1,16 +1,16 @@
-<odoo>
-    <data>
-      <record model="open_academy.course" id="course0">
-        <field name="name">Javascript Course</field>
-        <field name="description">Javascript Course</field>
-      </record>
-      <record model="open_academy.course" id="course0">
-        <field name="name">React Course</field>
-        <field name="description">React Course</field>
-      </record>
-      <record model="open_academy.course" id="course0">
-        <field name="name">Python Course</field>
-        <field name="description">Python Course</field>
-      </record>
-    </data>
-</odoo>
\ No newline at end of file
+<odoo noupdate="1">
+    <record id="course_demp_javascript" model="open_academy.course">
```

Comment:
typo "demp"

Reply:
Also you change the model name and the model is still open_academy.course, did you test it before asks for a review? 
this is wrong